src/birthder.js

```diff
@@ -44,11 +44,17 @@
   // Here are the OUTPUT format strings, they follow other rules;
   // See https://momentjs.com/docs/#/displaying/ for details.
   const OUTPUT_SHORT_DATE_FORMAT = 'DD.MM'
-  const OUTPUT_DATE_FORMAT = 'DD.MM.YYYY'
```

Comment:
Let's keep this constant:
- it is no need to delete it to refactor user sorting
- it was created to collect all date formats in one place.